source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -2,7 +2,7 @@
 # frozen_string_literal: true
 
 class CAdvisorMetricsAPIClient
-  require "yajl/json_gem"
+  require "json"
```

Comment:
since you are removing  require "yajl". please double check we dont have any references of using "yajl"

Reply:
done